lib/power-after-fail-lib.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Documentation       Collection of keywords related to the Power State After
+...                 Power Fail option
+
+Resource            ../keywords.robot
+
+
+*** Keywords ***
+Enter Power Management Options
+    [Documentation]    This keyword enters the Power Management Options
+    ...    from the Dasharo System Features submenu.
+    ${menu_construction}=    Get Setup SubMenu Construction
+    ${system_index}=    Get Index From List    ${menu_construction}    Power Management Options
+    Press Key N Times And Enter    ${system_index}    ${ARROW_DOWN}
+
+Set Power After Fail State
+    [Documentation]    Sets the Power State After Power Fail option to a
+    ...    desired state declared in ${state}. Available options are:
+    ...    Powered Off
+    ...    Powered On
+    ...    The state at the moment of failure
+    [Arguments]    ${state}
+    ${menu_construction}=    Get Setup SubMenu Construction
+    Select Option From List    ${menu_construction}    Power state after power    ${state}
+
+Simulate Power Failure
+    [Documentation]    This keyword simulates a power failure to the DUT.
+    Sonoff Power Off
+    Sleep    10s
+    Sonoff Power On
+    Sleep    5s
```

Comment:
Do not sleep here, otherwise robot may miss some strings if they are parsed right after this keyword. Also there is no need to sleep at all after restoring power.

Reply:
Fixed in [recent commit](https://github.com/Dasharo/open-source-firmware-validation/commit/e84e768780bae0482929f7309386c111a401257c)

Reply:
@filipleple my comment appplied only to the line 31: `Sleep    5s`. We need to give a couple of seconds before restoring the power so `Sleep    10s` between sonoff calls is desired and should be brought back.

Reply:
OK the 10s sleep [line](https://github.com/Dasharo/open-source-firmware-validation/commit/4e0c68c288a75411d2cdfa55e8cc1ba38cfaa79a#diff-ee78c06a5fbccca05b7cef40ff817bff0d7d9e5526ddf1f2127b69c97d91ac49R27) is brought back